projects/demo/src/modules/experimental/button-vertical/examples/import/insert-template.md

```diff
@@ -0,0 +1,9 @@
+```html
+<button
+  tuiButton
+  tuiButtonVertical
+>
+  <tui-svg src="tuiIconStarLarge"></tui-svg>
```

Comment:
After my PR is merged we should change most of the examples to just passing `iconLeft`